go.mod

```diff
@@ -29,11 +29,11 @@ require (
 	github.com/spf13/cobra v0.0.7
 	github.com/spf13/viper v1.6.3
 	github.com/vmihailenco/taskq v0.0.0-20190605141845-97870321dc66
+	github.com/vmihailenco/taskq/v3 v3.0.0-beta.8
```

Comment:
This should be 	github.com/vmihailenco/taskq/v3 v3.0.0-beta.8.0.20200427064301-df0ef147bc92

Reply:
This change is already present in beta.8 .